### DISCUSSION TITLE
Welcome to WaterMonitorDevice Discussions!

### DISCUSSION THREAD

**Discussion by digitalwaters-jason** [Digital-Waters/WaterMonitorDevice#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by atop8918** [comment]
Hi, this is Andrew. I do embedded systems design including hardware and software. I've been known to dabble in full stack development as well though my front end skills are crap. Also play guitar a bit. 
I love the ethos of Digital Waters: from the actual purpose of the hardware to the community-oriented and FOSS/H distribution model.
I prefer spaces to tabs though I can see the benefits of both.
Big fan of blue cheese but not mayo.
Don't ask me why I just started 2 flame wars.

**Reply by digitalwaters-jason** [reply]
There is enough space for both mayo and the various delicious kinds of blue cheese in the Digital Waters house 🤣

Welcome Andrew!
